lang/ui.ca.json

```
@@ -0,0 +1,1161 @@
+{
+  "about": {
+    "defaultMessage": "Quant a",
+    "description": "About action"
+  },
+  "about-comic": {
+    "defaultMessage": "Còmic de tres panells titulat \"MicroPython Rocks\" de Mike Rowbit. Una serp de dibuixos animats presenta a Damien, dient \"Et presento Damien... Va crear MicroPython\". Dues serps parlen de MicroPython. La serp groga diu \"MicroPython està dissenyat per funcionar en ordinadors molt petits\". \"Com la teva micro:bit de la BBC\" respon la serp morada.\" La serp groga continua \"Però Python pot executar-se a qualsevol lloc\". La serp morada hi està d'acord, dient \"Com en aquest bastidor de servidors que fan funcionar llocs de web enormes\". El fons darrere de les serps mostra un bastidor de servidors.",
+    "description": "Alt text for comic in the about dialog"
+  },
+  "about-microbit": {
+    "defaultMessage": "Fet amb amor per la <link>Micro:bit Educational Foundation and contributors</link>",
+    "description": "Text in about dialog"
+  },
+  "about-micropython": {
+    "defaultMessage": "L'editor depèn de <link>MicroPython</link>, fet per Damien George i una comunitat de desenvolupadors d'arreu del món.",
+    "description": "Text in about dialog"
+  },
+  "about-read-less-micropython": {
+    "defaultMessage": "Llegeix menys sobre MicroPython",
+    "description": "Aria label to collapse an expanded section about MicroPython"
+  },
+  "about-read-more-micropython": {
+    "defaultMessage": "Llegeix més sobre MicroPython",
+    "description": "Aria label to expand a collapsed section about MicroPython"
+  },
+  "added-change": {
+    "defaultMessage": "S'ha afegit el fitxer {changeName}",
+    "description": "Change made to file"
+  },
+  "api-description": {
+    "defaultMessage": "Per a l'ús i els exemples, consulta <link>Reference</link>(Referència)",
```

```suggestion
    "defaultMessage": "Per a l'ús i els exemples, consulta <link>Reference</link>(Referència)",
```

Not sure why this would be in brackets.